source/sass/components/publication-page.scss

```diff
@@ -50,8 +50,8 @@
 }
 
 .publication-chapter-number {
-  width: 161px;
-  height: 161px;
+  width: 160px;
```

Comment:
161px is a really weird number so I just changed it to 160px.

Reply:
Good catch. I likely copied this CSS directly from the design.